trulens_eval/tests/unit/test_embedding_feedback.py

```diff
@@ -0,0 +1,36 @@
+"""
+Test suites meant for testing the base functionality of the Embeddings feedback functions.
+"""
+
+import unittest
+from trulens_eval.feedback.embeddings import Embeddings
+import numpy as np
+from llama_index.embeddings.huggingface import HuggingFaceEmbedding
+
+class TestEmbeddings(unittest.TestCase):
+    def setUp(self):
+        self.embed_model = HuggingFaceEmbedding(model_name="BAAI/bge-small-en-v1.5")
+        self.embeddings = Embeddings(embed_model=self.embed_model)
+
+    def test_cosine_distance_type(self):
+        assert type(self.embeddings.cosine_distance("apple","banana")) == np.float64
```

Comment:
Use `self.assertIsInstance` instead as it gives better messages and technically can't be optimized away. In general you want to use the test asserting methods instead of `assert` for these reasons.